### PR TITLE
Add better logging

### PR DESCRIPTION
**Pull request by philgzl**
This PR implements better video progress logging. Currently only the progress of the current image generation is being logged. So it's hard to tell how far from completion the entire video generation is. Below is an example of how the logs look (could be different depending on how you set up `logging` in your script):
```
Fetching 12 files: 100%|██████████| 12/12 [00:00<00:00, 563.35it/s]
[20:31:11] [0/2] [0/13] Generating frames 0-3
100%|██████████| 50/50 [00:18<00:00,  2.71it/s]
[20:31:31] [0/2] [1/13] Generating frames 4-7
100%|██████████| 50/50 [00:13<00:00,  3.68it/s]
[20:31:45] [0/2] [2/13] Generating frames 8-11
100%|██████████| 50/50 [00:13<00:00,  3.67it/s]
...
[20:36:44] [1/2] [10/13] Generating frames 40-43
100%|██████████| 50/50 [00:13<00:00,  3.64it/s]
[20:36:59] [1/2] [11/13] Generating frames 44-47
100%|██████████| 50/50 [00:13<00:00,  3.64it/s]
[20:37:13] [1/2] [12/13] Generating frames 48-49
100%|██████████| 50/50 [00:07<00:00,  6.81it/s]
```
In this example I set up `logging` like this:
```python
logging.basicConfig(
    level=logging.INFO,
    format='[%(asctime)s] %(message)s',
    datefmt='%H:%M:%S',
)
```
The new log messages are formatted as
```
[<walk_index>/<num_walks>] [<batch_index>/<num_batches>] Generating frame(s) <frame indexes>
```
where `<walk_index>` and `<num_walks>` refer to walks between two prompts.

### CODE REVIEW

**Comment by Atomic-Germ** [comment]
This could address #116 and #147 nicely

**Comment by nateraw** [comment]
I was thinkin of ditching the underlying progress bar and making a new one, updating with info like this w/ one master progress bar...but this idea is much simpler. Nice!!!

Will take a look asap

**Comment by nateraw** [comment]
Sorry again for delay. will get to this as soon as I can. really really appreciate the contribution ❤️

**Comment by nateraw** [comment]
Really great PR, thank you :) Just added a line to logger init so it will show by default when running this without setting up logging in your script itself.

File: stable_diffusion_videos/stable_diffusion_pipeline.py [hunk 1]
```diff
@@ -764,6 +776,7 @@ def walk(
                 else None,
                 skip=skip,
                 negative_prompt=negative_prompt,
+                step=(i, len(prompts)-1),
```
Comment (nateraw) [inline stable_diffusion_videos/stable_diffusion_pipeline.py:779]:
```suggestion
                step=(i, len(prompts) - 1),
```

**Review by nateraw — APPROVED** [review]
LGTM

**Pull request merged**